backend/config.py

```
@@ -4,9 +4,76 @@
 This module handles configuration loading from environment variables and provides
 type-safe configuration objects.
 """
-from typing import List, Optional
+from typing import List, Optional, Literal, Dict, Any
```

Ruff (0.11.9)

`typing.Dict` imported but unused

Remove unused import

(F401)